src/languages/en.js

```diff
@@ -329,6 +329,7 @@ export default {
         payerSettled: ({amount}) => `settled up ${amount}`,
         noReimbursableExpenses: 'This report has an invalid amount',
         pendingConversionMessage: "Total will update when you're back online",
+        threadReportName: ({formattedAmount, comment}) => `${formattedAmount} request${comment ? ` for ${comment}` : ''}`,
```

Comment:
Should be coming through now
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/4073354/467dbbc4-590e-4595-9f3b-3788a15c8aad">